plugins/SimulationView/layers3d.shader

```diff
@@ -268,6 +275,29 @@ geometry41core =
 
             EndPrimitive();
         }
+
+
+        if ((u_show_starts == 1) && (v_prev_line_type[0] != 1) && (v_line_type[0] == 1)) {
```

Comment:
I'm not so sure about this one, it will do draw calls for a lot more start boxes but I think you'll only be able to see the inner wall start boxes on the topmost visible layer, because the rest is occluded by the outer walls.

Reply:
I think it's also fine without it. If people request it we can always consider adding it. I wouldn't block the PR on it since it already provides clear value :)